### PR TITLE
Acquisition session - Metadata File

### PR DESCRIPTION
**Pull request by Gouga34**

### CODE REVIEW

File: phis2-ws/src/main/java/phis2ws/service/dao/phis/ExperimentDao.java [hunk 1]
```diff
@@ -126,6 +147,63 @@ public Experiment get(ResultSet result) throws SQLException {
...
+            //Manipulates database results
+            while (queryResult.next()) {
+                Experiment experiment = new Experiment();
+                experiment.setUri(queryResult.getString(sqlFields.get("uri")));
+                experiment.setAlias(queryResult.getString(sqlFields.get("alias")));
+                experiment.setCropSpecies(queryResult.getString(sqlFields.get("cropSpecies")));
+                
+                experiments.add(experiment);
+            }
+
```
Comment (annetireau) [inline phis2-ws/src/main/java/phis2ws/service/dao/phis/ExperimentDao.java]: Please remove blank line

File: phis2-ws/src/main/java/phis2ws/service/dao/phis/ExperimentDao.java [hunk 2]
```diff
@@ -249,51 +327,50 @@ public Experiment get(ResultSet result) throws SQLException {
...
+            connection = dataSource.getConnection();
+            statement = connection.createStatement();
+            resultSet = statement.executeQuery(query.toString());
+
+            if(resultSet.next()) {
+                return resultSet.getInt(1);
+            }else{
+                return 0;
+            }
+
```
Comment (annetireau) [inline phis2-ws/src/main/java/phis2ws/service/dao/phis/ExperimentDao.java]: Please remove blank lines

File: phis2-ws/src/main/java/phis2ws/service/dao/phis/ExperimentDao.java (hunk 1 above)
Comment (Gouga34) [reply phis2-ws/src/main/java/phis2ws/service/dao/phis/ExperimentDao.java]: done

File: phis2-ws/src/main/java/phis2ws/service/dao/phis/ExperimentDao.java (hunk 2 above)
Comment (Gouga34) [reply phis2-ws/src/main/java/phis2ws/service/dao/phis/ExperimentDao.java]: done

**Pull request merged**